### PR TITLE
[vcpkg baseline][easyhook] Remove mismatch prebuild library after build

### PR DESCRIPTION
**Pull request by JackBoosY**
The prebuild library `AUX_ULIB_${ARCH}.lib` should match the triplet.
Remove the mismatch one.

### CODE REVIEW

File: scripts/ci.baseline.txt [hunk 1]
```diff
@@ -247,13 +247,6 @@ duilib:x64-uwp=fail
 duktape:x64-osx=skip
 
 eastl:arm-uwp=fail
-easyhook:arm64-windows=fail
-easyhook:arm-uwp=fail
```
Comment (JackBoosY) [inline scripts/ci.baseline.txt:251]: For arm and uwp, there is no following configuration in the upstream msbuild project files (.vcproj).

File: ports/easyhook/portfile.cmake [hunk 2]
```diff
@@ -17,20 +11,50 @@ vcpkg_from_github(
...
+    file(REMOVE "${CURRENT_PACKAGES_DIR}/debug/bin/EasyHook.dll" "${CURRENT_PACKAGES_DIR}/debug/bin/EasyHook.pdb")
+endif()
 
 # Install includes
-file(INSTALL ${SOURCE_PATH}/Public/easyhook.h DESTINATION ${CURRENT_PACKAGES_DIR}/include/easyhook)
+file(INSTALL "${SOURCE_PATH}/Public/easyhook.h" DESTINATION "${CURRENT_PACKAGES_DIR}/include/easyhook")
 
 # Handle copyright
-file(INSTALL ${SOURCE_PATH}/LICENSE DESTINATION ${CURRENT_PACKAGES_DIR}/share/easyhook RENAME copyright)
+file(INSTALL "${SOURCE_PATH}/LICENSE" DESTINATION "${CURRENT_PACKAGES_DIR}/share/easyhook" RENAME copyright)
```
Comment (PhoebeHui) [inline ports/easyhook/portfile.cmake:60]:
```suggestion
file(INSTALL "${SOURCE_PATH}/LICENSE" DESTINATION "${CURRENT_PACKAGES_DIR}/share/${PORT}" RENAME copyright)

```

Comment (JackBoosY) [inline ports/easyhook/portfile.cmake:60]:
```suggestion
file(INSTALL "${SOURCE_PATH}/LICENSE" DESTINATION "${CURRENT_PACKAGES_DIR}/share/${PORT}" RENAME copyright)

```

File: ports/easyhook/vcpkg.json [hunk 3]
```diff
@@ -0,0 +1,8 @@
+{
+  "name": "easyhook",
+  "version-string": "2.7.6789.0",
```
Comment (JackBoosY) [inline ports/easyhook/vcpkg.json:3]:
```suggestion
  "version": "2.7.6789.0",
```

File: versions/e-/easyhook.json [hunk 4]
```diff
@@ -1,5 +1,10 @@
 {
   "versions": [
+    {
+      "git-tree": "178522271d2755aedb74382d5177d7856ec66435",
+      "version-string": "2.7.6789.0",
```
Comment (JackBoosY) [inline versions/e-/easyhook.json:5]:
```suggestion
      "version": "2.7.6789.0",
```

**Review by PhoebeHui — APPROVED** [review]

File: versions/e-/easyhook.json [hunk 5]
```diff
@@ -1,5 +1,10 @@
 {
   "versions": [
+    {
+      "git-tree": "178522271d2755aedb74382d5177d7856ec66435",
```
Comment (JackBoosY) [inline versions/e-/easyhook.json:4]:
```suggestion
      "git-tree": "e427f42cec56130de81333fda405afabc88c9e82",
```

**Comment by BillyONeal** [comment]
Any ideas of what changed here? It's strange for a port last substantially edited 2 years ago to have a lint failure now.

**Review by BillyONeal — APPROVED** [review]

**Comment by BillyONeal** [comment]
Actually I think it might have been https://github.com/microsoft/vcpkg-tool/pull/126

**Comment by BillyONeal** [comment]
Yeah, this generates ".LIB", but we previously only checked ".lib": https://github.com/microsoft/vcpkg-tool/pull/126/files#diff-c7fac6e07e957eb3ff49dfce7c9e03deaa9bcab5d23e406a75e01a78b4c65685R563

**Pull request merged**